README.md

```diff
@@ -1,99 +1,124 @@
+===========================
 [![Go Report Card](https://goreportcard.com/badge/github.com/opentelekomcloud/kontainer-engine-driver-otc)](https://goreportcard.com/report/github.com/opentelekomcloud/kontainer-engine-driver-otc)
 ![GitHub](https://img.shields.io/github/license/opentelekomcloud/kontainer-engine-driver-otc)
 [![Build Status](https://zuul.otc-service.com/api/tenant/eco/badge?project=opentelekomcloud/kontainer-engine-driver-otc&pipeline=gate)](https://zuul.otc-service.com/t/eco/builds?project=opentelekomcloud%2Fkontainer-engine-driver-otc&pipeline=gate)
 ![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/opentelekomcloud/kontainer-engine-driver-otc?include_prereleases)
 
-This repository contains the Open Telekom Cloud CCE cluster driver for the rancher server.
+This repository contains the Open Telekom Cloud CCE cluster driver (a. k. a. the "Kontainer-Engine-Driver") for the [Rancher](https://www.rancher.com/) Enterprise Kubernetes Management platform.
 
 Frontend part: 
  - https://github.com/opentelekomcloud/ui-cluster-driver-otc
 
-## Building
+## Prerequisites
+
+This driver engine for the Open Telekom Cloud assumes that you have already the **cluster server** of the Rancher Enterprise Kubernetes management platform installed, either on one or several bare-metal-servers, in a Kubernetes cluster, or even in a single container. All options and a thorough discussion of all alternatives and options are covered by the [server installation manual](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade).
+
+This driver is tested with **Rancher 2.7.9**. Any newer version should do. You need to create at least one user on the Rancher cluster server, a password, and the Rancher server log-in URL. For details, refer to the Rancher manual.
 
-`make build`
+If you want to utilize **infrastructure resources provisioned by the Open Telekom Cloud**, you need exactly this very driver. The compute, network, and storage resources covered by the Open Telekom Cloud can be consumed and managed by the Rancher platform. You thus need **access credentials** for the Open Telekom Cloud. They come usually in form of a `clouds.yaml` file or an AK/SK pair.
 
-Will output driver binaries into the `bin` directory, these can be imported 
+Depending on your cluster setup, you may need **preconfigured network resources** such as a VPC, with at least one network, Internet connectivity for example via a NAT gateway or a router configuration associated with an EIP. You should upload an **SSH key pair** before you configure a new cluster. Please refer for all of this to your [Open Telekom Cloud documentation](https://docs.otc.t-systems.com/). These few tasks can be performed either via the [Open Telekom Cloud browser-based console](https://console.otc.t-systems.com/) or with a CLI tool like the [OpenStack Client](https://docs.openstack.org/python-openstackclient/).
```

Comment:
In future we will add possibility to create keypair from rancher ui, but ok for now.